### DISCUSSION TITLE
YimMenu

### DISCUSSION THREAD

**Discussion by mizkezz75** [YimMenu/YimMenu#2569] [General]
Is the menu safe? (will i get banned if i use it.) (15th of december 23)

**Comment by Yimura** [comment]
Have you checked if this was answered before recently in any other YimMenu discussions? I’m pretty sure it was.

**Reply by mizkezz75** [reply]
Old ones and yim was updated.

**Reply by Yimura** [reply]
It was answered here, took me 10 seconds to find #2520

**Reply by mizkezz75** [reply]
Oh :P sorry))

**Reply by Yimura** [reply]
I know it doesn’t 100% answers your question but the reasoning stays the same, it _should_ be safe but we won’t know 100% for sure until those 30 days have passed.

**Reply by mizkezz75** [reply]
No i get it, gotta do what you gotta do

**Reply by oMaLoKa** [reply]
o que ouve que o yim ta fechandoi o jogo sabem se ira demorar pra voltar ao normal ?
